SocialCareCaseViewerApi.Tests/V1/Boundary/Request/ListAllocationsRequestTests.cs

```diff
@@ -33,12 +33,14 @@ public void ValidationFailsWhenTwoOrMoreAttributesAreProvided()
             _request.MosaicId = 50;
             _request.WorkerId = 30;
             _request.WorkerEmail = "test@example.com";
+            _request.TeamId = 4;
```

Comment:
This is not really necessary for what the test is doing, but there's no harm adding it

Reply:
Yes, I was not sure whether to add it too, but it doesn't hurt, I guess.